### DISCUSSION TITLE
Questions about tag modification & tag preservation

### DISCUSSION THREAD

**Discussion by tcarroll2** [chicio/ID3TagEditor#96] [Q&A] [answered]
So, I cannot find an example of how to read a tag, modify a property within the tag (say comment), and write it back while preserving all other tag values in their original form.

Also, does ID3TagEditor play nicely with MusicBrainz Picard tags?  I guess my question is, does ID3TagEditor preserve the original tag properties if they are not modified?

Thanks for your work on this.  I'm just trying to figure out how to use your library to do what I need to do.

**Comment by chicio** [comment, marked as answer]
Ciao @tcarroll2 

at the moment is not possible to modify just one frame with the current ID3TagEditor api. I will develop it in the future but unfortunately right now I'm busy on other stuff (for some months).
To achieve you goal you can use the other ID3TagEditor apis:
* read the ID3 tag
* modify the frame that you want
* save back the ID3 tag

For the question "does ID3TagEditor play nicely with MusicBrainz Picard tags?", the answer is yes if they are ID3 tag as reported in the standard (https://id3.org), because ID3TagEditor is a library that strictly adhere to it (and it is reported in the implementations https://id3.org/Implementations)
